x/twasm/abci.go

```diff
@@ -57,7 +57,7 @@ func BeginBlocker(parentCtx sdk.Context, k abciKeeper, b abci.RequestBeginBlock)
 
 	msgBz, err := json.Marshal(msg)
 	if err != nil {
-		panic(err) // todo (reviewer): this will break consensus
```

Comment:
Wouldn't make sense to leave this, just as a warning?

Reply:
updated the text